controller/resource/metricsresource/resource.go

```diff
@@ -8,20 +8,11 @@ import (
 
 type Config struct {
 	Resource controller.Resource
-
-	// Name is name of the service using the controller. This may be the
-	// name of the executing operator or controller. The service name will
-	// be used to label metrics.
-	Name string
```

Comment:
This is in fact a breaking change. I would announce it again and people can go from there as it is simply about the deletion of one line.